erpnext_com/www/docs/user/manual/en/loan-management/loan-security.md

```diff
@@ -0,0 +1,42 @@
+<!-- add-breadcrumbs -->
+# Loan Security
+> Introduced in Version 13
+
+**A Loan Security is the actual security that is going to be pledged against a loan.**
+
+During a loan process, if the loan for which the customer is applying is a secured loan then some kind of security needs to be pledged against that loan. Loan Security specifies the actual security that is going to be pledged against a Loan
+
+<img class="screenshot" alt="Make Loan Security" src="{{docs_base_url}}/assets/img/loan-management/loan-security-flow.png">
+
+To access the Loan Security list, go to:
+> Home > Loan Management > Loan Security > Loan Security
+
+
+<img class="screenshot" alt="Loan Security" src="{{docs_base_url}}/assets/img/loan-management/loan-security.png">
+
+
+A Loan Security contains details about:
```

Comment:
this section and bullets are not needed since these are explained in the creation steps anyway. Remove.